robot-server/robot_server/protocols/dependencies.py

```diff
@@ -0,0 +1,23 @@
+"""Protocol router dependency wire-up."""
+from fastapi import Request
+from logging import getLogger
+from pathlib import Path
+from tempfile import gettempdir
+from .protocol_store import ProtocolStore
+
+log = getLogger(__name__)
+
+PROTOCOL_STORE_KEY = "protocol_store"
+PROTOCOL_STORE_DIRECTORY = Path(gettempdir()) / "protocols"
```

Comment:
```suggestion
PROTOCOL_STORE_DIRECTORY = Path(gettempdir()) / "opentrons-protocols"
```

IMO we should keep most of our filesystem stuff prefixed with `ot-` or `opentrons-` to distinguish it from system or vendor stuff.